src/core/client/admin/components/ModerateCard/ModerateCardContainer.tsx

```diff
@@ -262,17 +262,17 @@ const ModerateCardContainer: FunctionComponent<Props> = ({
 
   // Only highlight comments that have been flagged for containing a banned or
   // suspect word.
-  const highlight = useMemo(
-    () =>
-      comment.revision
-        ? comment.revision.actionCounts.flag.reasons
-            .COMMENT_DETECTED_BANNED_WORD +
-            comment.revision.actionCounts.flag.reasons
-              .COMMENT_DETECTED_SUSPECT_WORD >
-          0
-        : false,
-    [comment]
-  );
+  const highlight = useMemo(() => {
+    if (!comment.revision || !comment.revision.actionCounts) {
```

Comment:
`actionCounts` as per the schema cannot be null. If null is being passed, it could mean there's an issue with the resolver because the value for `actionCounts` is computed in the `CommentRevision` resolver. Please take a look there to see if there's an issue!

Reply:
Check the subscriptions are including the fragment!